touchforms/formplayer/migrations/0003_auto__add_session.py

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        
+        # Adding model 'Session'
+        db.create_table(u'formplayer_session', (
```

Comment:
at a minimum if we're not going to add this we could at least add a `last_modified` field so that we can later write a management command to delete anything that hasn't been touched in a long time